histograms/models.py

```diff
@@ -453,7 +457,11 @@ def from_nanodqm(file_path, data_era: str = "", read_chunk_lumi: int = -1):
         histogram_data_file, created = HistogramDataFile.objects.get_or_create(
             filepath=file_path
         )
-        histogram_data_file.data_dimensionality = HistogramDataFile.DIMENSIONALITY_2D
+        #histogram_data_file.data_dimensionality = HistogramDataFile.DIMENSIONALITY_2D
+        if histogram_data_file.data_dimensionality == HistogramDataFile.DIMENSIONALITY_UNKNOWN:
+            histogram_data_file.data_dimensionality = HistogramDataFile.DIMENSIONALITY_2D
+        elif histogram_data_file.data_dimensionality == HistogramDataFile.DIMENSIONALITY_1D:
+            histogram_data_file.data_dimensionality = HistogramDataFile.DIMENSIONALITY_1D2D
```

Comment:
I suppose `1D2D` is not intended, right?

Reply:
It is intended. This `1D2D` type should indicate that the file has 1D and 2D histograms parsed.